tests/unit/mixins/css-test.js

```diff
@@ -37,6 +37,13 @@ describe('Unit / Mixin / css', function () {
     it('should strip the junk from toString()', function () {
       expect(componentName).to.equal('css-component')
     })
+    it('should extract a className from a nested component', function () {
+      component.toString = function () {
+        return '<baz@component:deep/nested/component/foo-bar::id>'
+      }
```

Comment:
Can we maybe actually create a nested component and check that one? So then if `toString()` changes for those nested components we'll know about it here instead of in an app somewhere.